mlir/include/mlir/Dialect/Vector/IR/VectorOps.h

```diff
@@ -68,9 +68,13 @@ enum class BroadcastableToResult {
   DimensionMismatch = 2,
   SourceTypeNotAVector = 3
 };
+struct VectorDim {
+  int64_t dim;
+  bool scalableFlag;
```

Comment:
nit: I don't find this name very descriptive, `true`/`false` doesn't have a clear mapping to static/scalable. Maybe something like `isScalable` would be more fitting?